app/src/main/java/com/example/lapnguyen/weather_3/screen/mainscreen/MainPresenter.java

```diff
@@ -1,9 +1,25 @@
 package com.example.lapnguyen.weather_3.screen.mainscreen;
 
-import android.app.Activity;
+import android.Manifest;
```

Comment:
ERROR: Unused import: android.Manifest. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports